.ci/cico_updates_openshift.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2020 Red Hat, Inc.
+# Copyright (c) 2019-2021 Red Hat, Inc.
```

Comment:
We lose original creation date

Reply:
Sorry, but I thought we considered in the stand up to align all license headers to the same value, but after that developers should care about correct year for new files.